app.py

```diff
@@ -970,14 +983,33 @@ def handle_manage_schedule_option_button(ack, body, client, logger, context):
                         "text": "Select time",
                         "emoji": True
                     },
-                    "action_id": "event_time_select"
+                    "action_id": "event_start_time_select"
                 },
                 "label": {
                     "type": "plain_text",
                     "text": "Event Start",
                     "emoji": True
                 }
 		    },
+            {
+                "type": "input",
+                "block_id": "event_end_time_select",
+                "element": {
+                    "type": "timepicker",
+                    "initial_time": "06:30",
```

Comment:
Could make an argument for the default being 6:15, but that's nitpicky

Reply:
Yeah, it should be.  I missed this one while I was digging through all of the constants.  That's part of future things I'd like to explore - getting rid of some duplications through the code, so I can only add this a single time.